proto/babylon/zoneconcierge/v1/zoneconcierge.proto

```diff
@@ -103,22 +72,21 @@ message ProofEpochSealed {
   tendermint.crypto.ProofOps proof_epoch_val_set = 3;
 }
 
-// ProofFinalizedChainInfo is a set of proofs that attest a chain info is
+// ProofFinalizedConsumer is a set of proofs that attest a consumer is
 // BTC-finalised
-message ProofFinalizedChainInfo {
+message ProofFinalizedConsumer {
```

Comment:
- Let's stick to `BSN` and avoid using `Consumer` moving forward?
- Wdyt we call this `ProofFinalizedHeader` because the proof is still about the header?